packages/web/src/components/createCell.tsx

```diff
@@ -71,27 +71,17 @@ export type CellFailureProps<TVariables extends OperationVariables = any> = {
   updating?: boolean
 }
 
-// aka guarantee that all properties in T exist
-// This is necessary for Cells, because if it doesn't exist it'll go to Empty or Failure
-type Guaranteed<T> = {
-  [K in keyof T]-?: NonNullable<T[K]>
-}
-
 /**
  * Use this type, if you are forwarding on the data from your Cell's Success component
- * Because Cells automatically checks for "empty", or "errors" - if you receive the data type in your
- * Success component, it means the data is guaranteed (and non-optional)
  *
  * @params TData = Type of data based on your graphql query. This can be imported from 'types/graphql'
  * @example
  * import type {FindPosts} from 'types/graphql'
  *
  * const { post } = CellSuccessData<FindPosts>
  *
- * post.id // post is non optional, so no need to do post?.id
```

Comment:
This would only be true when _all_ of the following are true:
- `Empty` is defined
- `isEmpty` is not customized, _or_ its custom implementation retains the presence check (which is admittedly likely 🙂)
- `post` is the only field in your graphql query, _or_ all fields are present/non-empty